openmetadata-ui/src/main/resources/ui/src/components/Glossary/GlossaryDetails/GlossaryDetails.component.tsx

```diff
@@ -365,6 +365,7 @@ const GlossaryDetails = ({
         </Col>
         <Col span={24}>
           <Tabs
+            destroyInactiveTabPane
```

Comment:
What's the reason for adding this?

Reply:
Here for the Table we are calculating the width of column based on. table width, and after changing the Tab the grid library make the table size to 0. When we come back, the column width is then calculate on 0, by it was messing there. Now with desctorying the tab after even arrival it will see it as new table of table.

Reply:
Destroy is expensive can we look for something else?

Reply:
Have moved the glossary issue to this PR and removed from here to unblock this PR https://github.com/open-metadata/OpenMetadata/pull/20460